libs/ngworker/lumberjack-applicationinsights-driver/src/lib/log-drivers/lumberjack-applicationinsights-driver.ts

```diff
@@ -0,0 +1,129 @@
+import { Inject, Injectable } from '@angular/core';
+import { ApplicationInsights, SeverityLevel } from '@microsoft/applicationinsights-web';
+
+import { LumberjackLogDriver, LumberjackLogDriverLog, LumberjackLogPayload } from '@ngworker/lumberjack';
+
+import { LumberjackApplicationinsightsDriverInternalConfig } from '../configuration/lumberjack-applicationinsights-driver-internal.config';
+import { lumberjackApplicationinsightsDriverConfigToken } from '../configuration/lumberjack-applicationinsights-driver.config.token';
+
+@Injectable()
+export class LumberjackApplicationinsightsDriver<TPayload extends LumberjackLogPayload | void = void>
+  implements LumberjackLogDriver<TPayload> {
+  static driverIdentifier = 'LumberjackApplicationinsightsDriver';
+
+  constructor(
+    @Inject(lumberjackApplicationinsightsDriverConfigToken)
+    public config: LumberjackApplicationinsightsDriverInternalConfig,
+    private readonly appInsights: ApplicationInsights
```

Comment:
`ApplicationInsights` isn't decorated with `Injectable`. Do you normally use it this way?

Reply:
Usually I don't do it. Here I wanted to inject it like we injecting DOCUMENT, so in tests it can be replaced with mock. Anyway, we use factory to init this class and not direct DI.

Reply:
Ah, you're right. I see where you initialized it in the root driver module.